src/subcommands/subcommand.go

```diff
@@ -178,12 +161,12 @@ func inferSourceType(uris []string) (string, error) {
 	return sourceTypeStr, nil
 }
 
-func executeAdd(args []string) {
+func executeAdd(args []string, defaultIndexPath string) {
```

Comment:
we don't need defaultIndexPath to be args, you can make "$HOME/.semantifly/default" a constant local to the package eg ```defaultIndexPath```and just do
```
cmd.String("index-path", os.ExpandEnv(defaultIndexPath),...
```

Reply:
done